tests/UI/campaigns/functional/BO/02_orders/01_orders/createOrders/03_createSimpleOrderInBO.js

```diff
@@ -0,0 +1,125 @@
+const loginCommon = require('@commonTests/loginBO');
+
+// Import data
+
+// Customer
+const {DefaultAccount} = require('@data/demo/customer');
+
+// Products
+const {Products} = require('@data/demo/products');
+
+// Order status
+const {Statuses} = require('@data/demo/orderStatuses');
+
+// Carriers
+const {Carriers} = require('@data/demo/carriers');
+
+// Order to make data
+const orderToMake = {
+  customer: DefaultAccount,
+  products: [
+    {value: Products.demo_5, quantity: 4},
+  ],
+  deliveryAddress: 'Mon adresse',
+  invoiceAddress: 'Mon adresse',
+  deliveryOption: {
+    name: `${Carriers[1].name} - ${Carriers[1].delay}`,
+    freeShipping: true,
+  },
+  paymentMethod: 'Payments by check',
+  orderStatus: Statuses.paymentAccepted,
+
+  totalPrice: (Products.demo_5.price * 4) * 1.2, // Price tax included
+};
+
+// Import test context
+const testContext = require('@utils/testContext');
+
+const baseContext = 'functional_BO_orders_orders_createOrders_createSimpleOrderInBO';
+
+// Import expect from chai
+const {expect} = require('chai');
+
+let browserContext;
+let page;
+
+/*
+Go to create order page
+Search and choose a customer
+Add products to cart
+Choose address for delivery and invoice
+Choose payment status
+Set order status and save the order
+Check order status from view order page
+Check order total price from view order page
+ */
+describe('Create simple order in BO', async () => {
+  before(async function () {
+    browserContext = await helper.createBrowserContext(this.browser);
+    page = await helper.newTab(browserContext);
+  });
+
+  after(async () => {
+    await helper.closeBrowserContext(browserContext);
+  });
+
+  it('should login in BO', async function () {
+    await loginCommon.loginBO(this, page);
+  });
+
+  it('should go to orders page', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'goToOrdersPage', baseContext);
+
+    await dashboardPage.goToSubMenu(
+      page,
+      dashboardPage.ordersParentLink,
+      dashboardPage.ordersLink,
+    );
+
+    const pageTitle = await ordersPage.getPageTitle(page);
+    await expect(pageTitle).to.contains(ordersPage.pageTitle);
+  });
+
+  it('should go to create order page', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'goToCreateOrderPage', baseContext);
+
+    await ordersPage.goToCreateOrderPage(page);
+    const pageTitle = await addOrderPage.getPageTitle(page);
+    await expect(pageTitle).to.contains(addOrderPage.pageTitle);
+  });
+
+  describe('Choose which customer to order with', async () => {
+    it('should search for existent customer and check customer card', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'createOrder', baseContext);
+
+      await addOrderPage.createOrder(page, orderToMake);
+      const pageTitle = await viewOrderPage.getPageTitle(page);
+      await expect(pageTitle).to.contain(viewOrderPage.pageTitle);
+    });
+
```

Comment:
Could you check adresses are correct, products are correctly added and the payment method

Reply:
Be careful, the scope of this test must stay as minimal as possible. The more checks you have, the more risks of failure you add.
I'm all for checking and stuff but we will have some dedicated campaigns testing very thoroughly Orders, Products, Customers, Categories (all the sensible parts of the system).

Reply:
done